client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -331,6 +331,7 @@ class TransferDomainPrecheck extends React.Component {
 				<img
 					className="transfer-domain-step__illustration"
 					src={ '/calypso/images/illustrations/migrating-host-diy.svg' }
+					alt=""
```

Comment:
Why have an empty `alt` attribute here?

Reply:
That fixed an ESLint warning from an a11y rule. All images should have a textual `alt` description. Except those that serve only as a decoration and should be ignored by assistive technologies like screen readers. See also: https://www.w3.org/WAI/WCAG21/Techniques/html/H67

Reply:
Nice. Thanks for the link!